pkg/serverless/trace/inferredspan/span_enrichment_test.go

```diff
@@ -637,19 +637,35 @@ func TestEnrichInferredSpanWithEventBridgeEvent(t *testing.T) {
 	inferredSpan := mockInferredSpan()
 	inferredSpan.EnrichInferredSpanWithEventBridgeEvent(eventBridgeEvent)
 	span := inferredSpan.Span
-	assert.Equal(t, uint64(7353030974370088224), span.TraceID)
-	assert.Equal(t, uint64(8048964810003407541), span.SpanID)
```

Comment:
Why are these removed?

Reply:
Rey mentioned that we don't need to test them here because we overwrite them anyway